sass/layout/_sidebar-left.scss

```diff
@@ -1001,7 +1001,11 @@
         color: rgba(var(--sidebar-text-rgb), 0.6);
         list-style-type: none;
         opacity: 1;
+        transition-duration: 0.3s;
```

Comment:
This is twice as slow as the old transition. We should try this with the same 0.15s duration to see if it's still smooth at that rate, but if that doesn't work, we should confirm this with the UX team

Reply:
i have moved it back to 0.15s